docs/source/overview/quicktour.md

```diff
@@ -94,144 +94,63 @@ Now, let's take a look at the json configuration file you need to provide to opt
         "quantization": {
             "type": "OnnxDynamicQuantization"
         },
-    },
-
-    "engine": {
-        "log_severity_level": 0,
-        "evaluator": "common_evaluator",
-        "packaging_config": {}
     }
-}
 ```
-It is composed with 5 parts:
-### [Input Model](../overview/options.md/#input-model-information)
-You provide input model location and type. PyTorchModel, ONNXModel, OpenVINOModel and SNPEModel are supported model types.
-
-### [Systems](../overview/options.md/#systems-information)(Optional)
-You can define hardware target in this section for both Olive host system and target system used in the engine section below. Olive host system is for running optimizations and target system for evaluating the optimized model. The default value is local system with CPU. More details for the available systems can be found at [OliveSystems api reference](systems).
 
-### [Evaluator](../overview/options.md/#evaluators-information)
-You specify your performance requirements in evaluator, such as accuracy and latency, which the optimized candidate models should meet. Olive utilizes the information to tune the optimal set of optimization parameters for the "best" model.
+### Olive Engine
 
-### [Passes](../overview/options.md/#passes-information)
-An optimization technique is called as a Pass in Olive. You list optimizations that you want to apply on the input model. In this example, we first convert the pytorch model to ONNX then quantize it.
+Finally, you can select Olive features such as which performance metrics you want to use in this run, verbosity level etc.
 
-### [Engine](../overview/options.md/#engine-information)
-The engine tunes optimization passes to produces optimized model(s) based on evaluation criteria. It has default auto-tuning algorithm, also allow you to set your own. You can also define host system and target system in this section if you have to run the optimizations and evaluate the model on different hardware target. Please check [Engine api reference](engine) for more details.
+```json
+    "engine": {
+        "log_severity_level": 0,
+        "evaluator": "my_evaluator"
+    }
+```
 
-Note: In addition to these five core sectors, Olive provides a rich selection of optional configurations to suit diverse scenarios. For detailed information on these options, please refer to the [options.md](../overview/options.md/) file.
+Now you have a complete json file that you can use to accelerate the resnet model. For more detailed information about all the features supported by Olive, please refer to the [Olive Options](../overview/options.md) and Tutorials.
 
-## Olive Optimization Result
```

Comment:
Where can user get such info? At lease we need to link to the place in the quick tour where Olive output is explained, since there are multiple components user need to understand.

Reply:
- The Bert Quickstart example should cover one E2E flow.
- We should have how-to docs for each features individually. Olive has many features Auto-tuning, AML integration. Search Strategy, Azure/Docker Systems, Distributed Evaluators, Different optimization techniques, Remote Targets, Output Packaging etc.